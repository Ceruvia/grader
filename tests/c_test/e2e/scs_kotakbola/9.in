44 816 252
M 75
M 85
B 29
E
