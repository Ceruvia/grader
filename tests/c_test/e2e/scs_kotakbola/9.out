0 160 29
Ya
